content/en/blog/grpc-csharp-future.md

```diff
@@ -1,193 +1,82 @@
+the name of its nuget package.
+
+In September 2019, we [announced the general availability][ga] of
+**[grpc-dotnet][]**, a new **pure-C#** implementation of gRPC for .NET Core.
+
+{{% alert title="Note" color="info" %}}
+  For a detailed comparison of these two implementations, see [grpc-dotnet and
+  Grpc.Core comparison][comparison].
+
+  [comparison]: https://github.com/grpc/grpc-dotnet/blob/master/doc/implementation_comparison.md
+{{% /alert %}}
+
+Since then, grpc-dotnet has continued to mature and significantly increase
+both its feature set and user-base.
+
+We've reached a point where we can no longer justify the maintenance costs of
+two implementations of gRPC for C#. The Grpc.Core implementation is based on
+design tradeoffs that made sense in 2016, but are no longer optimal -- this
+makes it a good candidate for a progressive retirement.
+
+In contrast, the modern design of grpc-dotnet is the best choice for the
+community that it serves, given its seamless integration with .Net: as a pure C#
+implementation, it will be easier to use and debug. This also makes it more
+contribution-friendly for C# community members.
+
+## Phase-out plan {#plan}
+
+### Phase 1: maintenance-only
 
 **When: Effective immediately (May 2021)**
 
-From now on, we are no longer going to provide new features or enhancements for
-Grpc.Core. Important bugs and security issues will continue to be addressed in a
-normal way.
+Effective immediately, Grpc.Core is in maintenance-only mode: we won't provide
+Grpc.Core with new features or enhancements. Important bugs and security issues
+will continue to be addressed in a normal way.
 
 We will publish Grpc.Core releases normally, with the usual 6 weekly cadence.
 
 The releases will be based on the newest grpc C core native library build, so
 all new features that don't require C# specific work will also be included.
 
-**Phase 2: Grpc.Core becomes "Deprecated"**
+### Phase 2: deprecation
 
 **When: 1 year from now (May 2022)**
 
-Once this milestone is reached, Grpc.Core will no longer be officially supported
-and all the users will be strongly advised to only use grpc-dotnet from this
-point onwards.
-
-The Grpc.Core nuget packages will remain available in the nuget.org repository,
-but no more fixes will be provided (= not even security fixes).
-
-**Future of Grpc.Tools and Grpc.Core.Api nuget packages**
+Grpc.Core is deprecated: once this milestone is reached, Grpc.Core will no
+longer be officially supported. All the users will be strongly advised to
+transition to grpc-dotnet.
 
-Both packages will continue to be fully supported, since they are strictly
-speaking not part of Grpc.Core and they are also used by grpc-dotnet.
+The Grpc.Core nuget packages will remain available in the [nuget.org][]
+repository, but no more fixes will be provided -- not even security fixes.
 
-- the Grpc.Tools nuget package which provides the codegen build integration for
```

Comment:
I don't think not everyone is so familiar with the names of the nuget packages "Grpc.Tools" and "Grpc.Core.Api" to be able immediately recall what they're used for, so that's why I wanted summarize what are these package for in the text - can we put back the descriptions of those packages?